test/io.jl

```diff
@@ -11,37 +11,55 @@ module TestIO
                     E = CategoricalArray(["a", missing, "c", "d"]),
                     F = Vector{String}(undef, 4)
                     )
-        str = """
+        ioc = IOContext(IOBuffer(), :displaysize => (10, 10), :limit => false)
+        show(ioc, MIME(Symbol("text/latex")), df)
+        str = String(take!(ioc.io))
+        @test str == """
             \\begin{tabular}{r|cccccc}
             \t& A & B & C & D & E & F\\\\
             \t\\hline
             \t& $(Int) & String & LaTeXStr… & Float64⍰ & Categorical…⍰ & String\\\\
             \t\\hline
             \t1 & 1 & \\\$10.0 & \$\\alpha\$ & 1.0 & a & \\#undef \\\\
-            \t2 & 2 & M\\&F & \$\\beta\$ & 2.0 &  & \\#undef \\\\
-            \t3 & 3 & A\\textasciitilde{}B & \$\\gamma\$ &  & c & \\#undef \\\\
+            \t2 & 2 & M\\&F & \$\\beta\$ & 2.0 & missing & \\#undef \\\\
+            \t3 & 3 & A\\textasciitilde{}B & \$\\gamma\$ & missing & c & \\#undef \\\\
+            \t4 & 4 & \\textbackslash{}\\textbackslash{}alpha & \$\\sum_{i=1}^n \\delta_i\$ & 3.0 & d & \\#undef \\\\
+            \\end{tabular}
+            """
+    
+        ioc = IOContext(IOBuffer(), :displaysize => (10, 10), :limit => false)
+        show(ioc, MIME(Symbol("text/latex")), df, missingstring="missing")
+        str = String(take!(ioc.io))
+        @test str == """
+            \\begin{tabular}{r|cccccc}
+            \t& A & B & C & D & E & F\\\\
+            \t\\hline
+            \t& $(Int) & String & LaTeXStr… & Float64⍰ & Categorical…⍰ & String\\\\
+            \t\\hline
+            \t1 & 1 & \\\$10.0 & \$\\alpha\$ & 1.0 & a & \\#undef \\\\
+            \t2 & 2 & M\\&F & \$\\beta\$ & 2.0 & missing & \\#undef \\\\
+            \t3 & 3 & A\\textasciitilde{}B & \$\\gamma\$ & missing & c & \\#undef \\\\
             \t4 & 4 & \\textbackslash{}\\textbackslash{}alpha & \$\\sum_{i=1}^n \\delta_i\$ & 3.0 & d & \\#undef \\\\
             \\end{tabular}
             """
-        @test repr(MIME("text/latex"), df) == str
     end
 
     @testset "Huge LaTeX export" begin
         df = DataFrame(a=1:1000)
         ioc = IOContext(IOBuffer(), :displaysize => (10, 10), :limit => false)
-        show(ioc, "text/latex", df)
+        show(ioc, MIME(Symbol("text/latex")), df)
```

Comment:
`Symbol` isn't needed AFAICT.

Reply:
My point was that it should be just `"text/latex"` if we fix the signatures.